Fix Admin references in PE view docs- except user-management related segments (#1721)
wso2/docs-choreo-dev: en/pe-docs/docs/administer/inviting-members.md

## Inviting users

!!! note

>      - You must have **Send Invitations** or **Manage Invitations** permission under **USER-MANAGEMENT** Permission group to invite users. 

Reviewer: ```suggestion
     - You must have **Send Invitations** or **Manage Invitations** permission under **USER-MANAGEMENT** permission group to invite users. 
```

> A platform engineer can invite users to the organization by assigning them specific groups. Invited users receive an invitation via email. An invited user must accept the invitation to join the organization and access the resources of that organization.

Reviewer: ```suggestion
Platform engineers can invite users to the organization by assigning them specific groups. Invited users will receive an invitation via email. An invited user must accept the invitation in order to join the organization and access the resources.
```
